BSDmakefile

```diff
@@ -1,5 +1,5 @@
 #
-#  Copyright (C) CFEngine AS
+#  Copyright 2017 Northern.tech AS
```

Comment:
Oh, and retaining copyright for CFEngine is definitely wrong, the company doesn't exist anymore. All its copyrights were transferred to Northern.tech.